index.js

```diff
@@ -24,6 +24,10 @@ const XETHRU_THRESHOLD_MILLIS = 60 * 1000
 const WATCHDOG_TIMER_FREQUENCY = 60 * 1000
 
 const locationsDashboardTemplate = fs.readFileSync(`${__dirname}/locationsDashboard.mst`, 'utf-8')
+const sensorDashboardTemplate = fs.readFileSync(`${__dirname}/sensorDashboard.mst`, 'utf-8')
+const navPartial = fs.readFileSync(`${__dirname}/navPartial.mst`, 'utf-8')
+const landingCSSPartial = fs.readFileSync(`${__dirname}/landingCSSPartial.mst`, 'utf-8')
+const locationCSSPartial = fs.readFileSync(`${__dirname}/locationCSSPartial.mst`, 'utf-8')
```

Comment:
It actually goes with `sensorDashboardTemplate` which is the landing page so good point. `locationsCSSPartial` goes with `locationsDashboardTemplate` 😅 I was trying to avoid big long names but I think that maybe won't be possible if we want it to be very unambiguous 🤔 My inclination would be to rename `sensorDashboardTemplate` to `landingPageTemplate` I think which might avoid that though